src/fastapi_poe/types.py

```diff
@@ -61,6 +79,95 @@ class QueryRequest(BaseRequest):
     logit_bias: Dict[str, float] = {}
     stop_sequences: List[str] = []
 
+    _pending_tasks: Set[asyncio.Task] = set()
+    _attachment_upload_url = (
+        "https://www.quora.com/poe_api/file_attachment_3RD_PARTY_POST"
+    )
+
+    async def post_message_attachment(
```

Comment:
This doesn't feel appropriate for `types.py`. Could we make it a top-level function in `base.py` instead?